crates/evm/core/src/backend/mod.rs

```diff
@@ -1857,7 +1858,7 @@ pub(crate) fn merge_account_data<ExtDB: DatabaseRef>(
     // need to mock empty journal entries in case the current checkpoint is higher than the existing
     // journal entries
     while active_journaled_state.journal.len() > target_fork.journaled_state.journal.len() {
-        target_fork.journaled_state.journal.push(Default::default());
+        // target_fork.journaled_state.journal.push(Default::default());
```

Comment:
This must be the source of the tests / repros stalling on multiforking, looking into this

Reply:
Related: https://github.com/bluealloy/revm/pull/2440/files